bedrock/base/urlresolvers.py

```diff
@@ -93,26 +111,15 @@ def get_language(self):
 
     def get_best_language(self, accept_lang):
         """Given an Accept-Language header, return the best-matching language."""
-        LUM = settings.LANGUAGE_URL_MAP
-        langs = dict(LUM.items() + settings.CANONICAL_LOCALES.items())
-        # Add missing short locales to the list. This will automatically map
-        # en to en-GB (not en-US), es to es-AR (not es-ES), etc. in alphabetical
-        # order. To override this behavior, explicitly define a preferred locale
-        # map with the CANONICAL_LOCALES setting.
-        langs.update((k.split('-')[0], v) for k, v in LUM.items() if
-                     k.split('-')[0] not in langs)
-        try:
-            ranked = parse_accept_lang_header(accept_lang)
-        except ValueError:  # see https://code.djangoproject.com/ticket/21078
```

Comment:
I'm glad we don't have to catch this exception any more. I remember thinking how it would be forever before we could remove this line, but it only took 2 years.